modules/Kconfig.tfm

```diff
@@ -0,0 +1,17 @@
+# TF-M/PSA Related
+
+# Copyright (c) 2019 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config BUILD_WITH_TFM
+	bool "Build with TF-M for Secure Execution"
+
+config PSA_ARCH_API_TEST
```

Comment:
Seems like PSA_ARCH_API_TEST and PSA_API_TEST_SUITE should just be if we're building the psa sample app.

Also wonder if those should move under tests/ instead of samples

Reply:
I agree these should be moved to the app level, but I don't think the app belongs under `tests` myself, and should probably stay as an individual sample app. They aren't really unit tests in the Zephyr sense, but rather something you need to run once to generate a report (for a specific board) that you send out to a test lab to have your board certified as being PSA FF API compliant. The sample app is provided as a convenience to generate the required report when certifying a board with Zephyr, not to test the API itself on a regular basis.